packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1010,7 +1024,10 @@ export class GarbageCollector implements IGarbageCollector {
         this.updateStateSinceLastRun(gcData, logger);
 
         // Update the current state and update the runtime of all routes or ids that used as per the GC run.
-        this.updateCurrentState(gcData, gcResult, currentReferenceTimestampMs);
+        this.gcDataFromLastRun = cloneGCData(gcData);
```

Comment:
Why move these 3 initializations here?

Reply:
I couldn't come up with a better name for the 3 initializations

Reply:
Moving them back in and calling it `updateCurrentMarkState`